src/lib/plugins/moderation.js

```diff
@@ -132,7 +134,7 @@ module.exports.server = function (serv, settings) {
         if (player.op) {
           player.op = false
 
-          player.chat(`§7§o[Server: Deopped ${params[0]}]`)
+          player.chat(`§7§o[${player.username ?? 'Server'}: Deopped ${params[0]}]`)
```

Comment:
It might be unfavourable for the deopped target to know who the deopper is.

Reply:
Ok